uitools/toolkitcpp/import/Esri/ArcGISRuntime/Toolkit/MediaPopupElementView.qml

```diff
@@ -37,9 +36,90 @@ ColumnLayout {
     property real mediaMargin: 10
     property real imageTextMargin: 5
     property real layoutSpacing: 0
+    property bool isHoverable: true
+    property bool isMobile: (Qt.platform.os === "android" || Qt.platform.os === "ios")
 
     property var controller: null
 
+    Dialog {
+        id: fullScreenImageDialog
+        modal: true
+        visible: false
+        width: Overlay.overlay ? Overlay.overlay.width : 0
+        height: Overlay.overlay ? Overlay.overlay.height : 0
+        anchors.centerIn: Overlay.overlay
+
+        property string imageTitle: ""
+        property string imageCaption: ""
+        property var modelData: null
+        property var mediaType: null
+
+        ColumnLayout {
+            anchors.fill: parent
+            Layout.alignment: Qt.AlignHCenter
+
+            Label {
+                visible: fullScreenImageDialog.imageTitle !== ""
+                text: fullScreenImageDialog.imageTitle
+                font.weight: Font.Bold
+                color: "black"
```

Comment:
do these hardcoded colors work in light/dark mode?

Reply:
good catch!